cfme/tests/test_replication.py

```diff
@@ -329,7 +340,55 @@ def test_replication_global_region_dashboard():
             1.
             2. Dashboard on the Global displays data from the Remote region
     """
-    pass
+    remote_app, global_app = setup_replication
+    remote_provider = provider_app_crud(RHEVMProvider, remote_app)
```

Comment:
Any reason to choose `RHEVMProvider`? How about simply `InfraProvider` or something of that effect, which will be less tied to one type of provider.